browser/brave_drm_tab_helper.cc

```diff
@@ -92,14 +98,20 @@ void BraveDrmTabHelper::DidStartNavigation(
 
 void BraveDrmTabHelper::OnWidevineKeySystemAccessRequest() {
   is_widevine_requested_ = true;
+#if BUILDFLAG(IS_ANDROID)
```

Comment:
what is the point of this buildflag? It's false in both cases

Reply:
I forgot to change the second one back to `true`.
I only have Android and Linux. I'll find someone to test other platforms.